doc/guides/modules.rst

```diff
@@ -82,6 +82,26 @@ Each project in the ``west list`` output is tested like this:
        cmake: .
        kconfig: Kconfig
 
+- To execute both tests and samples available in modules the Zephyr test runner
+  (sanitycheck) should be pointed to the directories containing those samples and
+  tests. This can be done by specifying the path to both samples in tests in the
+  :file:`zephyr/module.yml` file, for example:
+
+  .. code-block:: yaml
+
+      build:
+        cmake: .
+      samples:
+        path: samples
+      tests:
+        path: tests
+      boards:
+        path: boards
```

Comment:
Now, the existing configuration is actually implying a path:
`cmake: <path>` and `kconfig: <path/Kconfig-file>` under the build, so I kind of feel strange that `path` is a key by itself.

How about:
```suggestion
      samples: <path/to/samples>
      tests: <path/to/tests>
      boards: <path/to/boards>
```
where the path is relative from beginning of the module, which would most often be:
```
      samples: samples
      tests: tests
      boards: boards
```

Of course we might want an additional level besides `build`, as example `source` ? (feel free to provide better name)

Reply:
@tejlmand 
That's because you can have multiple `samples:` and `tests:` but you can't have multiple CMake or Kconfig files.

Reply:
@tejlmand @nashif @mbolivar my proposal would be to accept multiple entries for `path:`:

```
samples:
  path: path/to/sample/one
  paht: path/to/sample/two
```

Reply:
Or, as @nashif mentioned perhaps:

```
samples:
  - path/to/sample/one
  - path/to/sammple/two
```

Reply:
Doing it as a list is the right way YAML-wise.